shared/i18n/locales/cs_CZ/translation.json

```diff
@@ -655,6 +655,7 @@
   "Sorry, you can only send {{MAX_INVITES}} invites at a time": "Litujeme, můžete odeslat pouze {{MAX_INVITES}} pozvánek najednou",
   "Invited members will receive access to": "Pozvaní členové obdrží přístup k",
   "{{collectionCount}} collections": "{{collectionCount}} collections",
+  "non-private collections when added": "non-private collections when added",
```

Comment:
What's the process for translating the newly added texts?

Reply:
Did you manually update these files? These translation json other than `en_US` are handled through the translation provider (CrowdIn)